app/concepts/matestack/ui/core/iframe/iframe.rb

```diff
@@ -0,0 +1,5 @@
+module Matestack::Ui::Core::Iframe
+  class Iframe < Matestack::Ui::Core::Component::Static
+
```

Comment:
Missing a lot of the attributes an iframe tag can have ([see reference](https://www.w3schools.com/tags/tag_iframe.asp)). See youtube or image component for reference on how to include multiple attributes into the `@tag_attributes`!